documents/motions/01.2025.1 - Spirit.md

```diff
@@ -1,14 +1,14 @@
 # SUBMISSION OF PROPOSED MOTION
 
-**Motion number:** 1.2021.1  
+**Motion number:** 1.2025.1  
 **Subject:** Spirit  
 **Intent:** Definition of the Spirit of the World Cube Association  
 **Submitted by:** WCA Board  
-**Date:** February 28, 2021  
+**Date:** March 1, 2025
 
 # Motion
 
 1. The World Cube Association (WCA) is the world governing body for the sport of Speedcubing.
 2. The Mission of the World Cube Association is to have more competitions in more countries with more people and more fun, under fair and equal conditions.
 3. The Spirit of the World Cube Association is that people from all over the world have fun together in a friendly atmosphere, help each other, and behave in a sportsmanlike fashion.
-4. The World Cube Association exists for its Registered Speedcubers and thus the key value of WCA Staff is to serve the Registered Speedcubers of the WCA. See [WCA Motion : Registered Speedcubers](wcadoc{documents/motions/07.2024.1 - Registered Speedcubers.pdf}).
+4. The World Cube Association exists for its Registered Speedcubers and thus the key value of WCA Volunteers is to serve the Registered Speedcubers of the WCA. See [WCA Motion : Registered Speedcubers](wcadoc{documents/motions/07.2024.1 - Registered Speedcubers.pdf}).
```

Comment:
```suggestion
4. The World Cube Association exists for its Registered Speedcubers and thus the key value of WCA Volunteers is to serve the Registered Speedcubers of the WCA. See [WCA Motion : Registered Speedcubers](wcadoc{documents/motions/07.2025.1 - Registered Speedcubers.pdf}).
```
since the referenced motion will also be updated